examples/cfq/input_pipeline.py

```diff
@@ -0,0 +1,210 @@
+# Copyright 2020 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""CFQ input pipeline."""
+
+# pylint: disable=too-many-arguments,import-error,too-many-instance-attributes,too-many-locals
+from typing import Dict, Text
+
+from absl import logging
+import tensorflow.compat.v2 as tf
+import tensorflow_datasets as tfds
+import tensorflow_text as text
+import jax.numpy as jnp
+
+import utils
+import preprocessing
+import constants
+
+ExampleType = Dict[Text, tf.Tensor]
+
+
+class CFQDataSource:
+    """Provides CFQ data as pre-processed batches, a vocab, and embeddings."""
```

Comment:
It seems you changed the indentation to 4 spaces, could you please change it back to 2 spaces everywhere?

Reply:
it seems to be 4 spaces in the style guide https://google.github.io/styleguide/pyguide.html#34-indentation why should 2 be used?